Source/WebInspectorUI/UserInterface/Views/ScopeBar.js

```diff
@@ -146,8 +145,18 @@ WI.ScopeBar = class ScopeBar extends WI.NavigationItem
         }
     }
 
+    _multipleItemSelectionDidChange(event)
+    {
+        this.dispatchEventToListeners(WI.ScopeBar.Event.SelectionChanged);
```

Comment:
this will likely also need to clear the selection from all other `WI.ScopeBarItem`